wasm/solana-cli/Cargo.toml

```diff
@@ -29,9 +29,9 @@ solana-cli-output-wasm = { path = "../utils/solana-cli-output" }
 solana-client-wasm = { path = "../solana-client" }
 solana-extra-wasm = { path = "../utils/solana-extra" }
 solana-playground-utils-wasm = { path = "../utils/solana-playground-utils" }
-solana-remote-wallet = { version = "=1.14.16", default-features = false }
+solana-remote-wallet = { version = ">=1.14, <1.17", default-features = false }
 solana-sdk = "*"
-solana-version = "=1.14.16"
```

Comment:
Specifying dependencies to be as general as possible is helpful for libraries like `anchor-lang` where a lot of people depend on it but currently `solana-cli-wasm` is only being used to generate a WASM binary(no code depends on it), which means we can, and should be, as specific as possible with our dependencies.